docs/services/firecrest.md

```diff
@@ -107,75 +142,241 @@ To use your Application to access FirecREST, follow the [API documentation](http
 One way to get started is by using [pyFirecREST](https://pyfirecrest.readthedocs.io/), a Python package with a collection of wrappers for the different functionalities of FirecREST.
 This package simplifies the usage of FirecREST by making multiple requests in the background for more complex workflows as well as by refreshing the access token before it expires.
 
+#### Try version 2
```

Comment:
Instead of having these as sub-headings, you can use "tabs" (just a suggestion)

https://squidfunk.github.io/mkdocs-material/reference/content-tabs/

You removed the a tab above, so maybe you are not a fan of how they look or work.

There are a few spots in these docs where you might want to use tabs for version1 and version 2, for consistency.
You could make version 2 the first tab, so it is what is displayed by default.

Reply:
> Instead of having these as sub-headings, you can use "tabs" (just a suggestion)
I like the idea.

> You removed the a tab above, so maybe you are not a fan of how they look or work.
I removed it because the option of migrate a client is not longer supported, thus there's just one option.